jon-nona/hack-assembler-typescript
# feat(line-parser): strip comments remove blank lines

# Pull Request

## Scope

HA-2: line parser - strip comments and remove blank lines

## Work Done

- add utils to read lines, strip comments and blank lines and re-assemble for output
- add tests

## Steps to Test

- create a text file with single and multi line comments
- pull the branch, build the app, run assemble on your assembly file
- it should be output with the empty lines and comments removed.
